secant/Models/ValidationWord.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ValidationWord.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 12.05.2022.
+//
+
+import Foundation
+
+/// Represents the data of a word that has been placed into an empty position, that will be used
+/// to validate the completed phrase when all ValidationWords have been placed.
+struct ValidationWord: Equatable {
```

Comment:
I don't think this is used elsewhere other than the phrase validation flow. Is it really an application-wide model?